docs/nodes/full-storage-node.mdx

```diff
@@ -88,50 +92,84 @@ A note on ports:
   after the IP address or use the  `--core.grpc.port` flag to specify another
   port if you prefer.
 
-````mdx-code-block
-<Tabs groupId="network">
-<TabItem value="mocha" label="Mocha">
-
 ```sh 
-celestia full start --core.ip <ip-address>
+celestia full start --core.ip <ip-address>:<port>
 ```
 
-</TabItem>
-<TabItem value="arabica" label="Arabica 🏗️">
+If you would like to find example RPC endpoints, check out the list of
+resources [here](../mocha-testnet).
```

Comment:
[here](./mocha-testnet)

Reply:
yo soy confused! what do you mean?

Reply:
updated comment

Reply:
Actually not sure if this makes a difference given you're using a certain style for links.

Reply:
😅🫡